examples/external-data/src/mock-customer-service/service.ts

```diff
@@ -288,16 +338,24 @@ export async function initializeCustomerService(props: ServiceProps): Promise<Se
 
 		if (eventType === "session-end") {
 			const typedRequest = request as SessionEndEventsListenerRequest;
-			const containerUrl = typedRequest.body?.containerUrl;
-			if (containerUrl === undefined || typeof containerUrl !== "string") {
-				const errorMessage = `Missing or malformed containerUrl: ${containerUrl}`;
+			const documentId = typedRequest.body?.documentId;
+			if (documentId === undefined || typeof documentId !== "string") {
+				const errorMessage = `Missing or malformed documentId: ${documentId}`;
+				result.status(400).json({ message: errorMessage });
+				return;
+			}
+
+			const tenantId = typedRequest.body?.tenantId;
+			if (tenantId === undefined || typeof tenantId !== "string") {
+				const errorMessage = `Missing or malformed tenantId: ${tenantId}`;
 				result.status(400).json({ message: errorMessage });
 				return;
 			}
 
 			// Removes the mapping of the given container URL from all task id's
-			const emptyTaskListRegistrationIds =
-				clientManager.removeAllClientTaskListRegistrations(containerUrl);
+			const emptyTaskListRegistrationIds = clientManager.removeAllClientTaskListRegistrations(
+				{ TenantId: tenantId, DocumentId: documentId },
+			);
```

Comment:
Let's double check this; when we were passing a string, that's a safe thing to use as a key in a Map. A record/object probably isn't; every instance of it will be a different object so retrieving from the map won't work. Passing them as a record is fine, but ClientManager might need to concat them into a string `<tenantId>_<documentId>` so it keeps using a string for the key locally.

Reply:
Added conversions everywhere to make the key a string!